provisioning/templates-api/kubernetes/templates/api/deployment.yaml

```diff
@@ -31,15 +31,15 @@ spec:
           resources:
             requests:
               cpu: "200m"
-              memory: "512Mi"
+              memory: "768Mi"
             limits:
-              memory: "512Mi"
+              memory: "768Mi"
```

Comment:
Chodia nam warning ohladom `Templates API` memory.
Vacsinou sa to drzi na `200MB`, ale sem tam to prekroci aj `500MB`.

Mal by tomu pomoct refactoring local/remote state.
Ak nie tak sa to bude profilovat.

Kym sa tak stane, tak by som tu pamat mierne zvysil.